sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -186,7 +186,14 @@ public function clear_status() {
 
 	private function get_status_option( $option ) {
 		$prefix = self::STATUS_OPTION_PREFIX;
-		return intval( get_option( "{$prefix}_{$option}", 0 ) );
+
+		$value = get_option( "{$prefix}_{$option}", null );
+		
+		if ( ! $value ) {
+			return null;
+		}
+
+		return intval( $value );
```

Comment:
I removed it as a simplification, but didn't realise that API consumers were looking for null instead of 0.